### ISSUE TITLE
[FEATURE REQUEST] To add the same save capability and create sessions/profiles as you have built in qpwgraph.

### ISSUE THREAD

**Issue by toadzrock** [rncbc/qjackctl#215]
As the title says, in qpwgraph the user can save profiles of connections, a very useful capability particularly for those of us that build large collections of apps to use at once with the inevitable forest of connections.

Can we please have this in qjackctl as well? The current patchbay setup falls short of this, and having a simple save, with naming rights, ala qpwgraph, would be simpler to use.

**Comment by tkna91** [comment]
Isn't that the session function of qjackctl?
![image](https://github.com/user-attachments/assets/2f80d672-15a2-4211-be94-b8f6390dd824)

**Comment by rncbc** [comment]
no, it's the "Patchbay" and not "Session" the most similar to qpwgraph's Patchbay functionality.

however they are not the same model.

and no there are no plans to bring one into another; besides you ought to use only one at any one time, unless total connection mayhem is desired ;)
